wp/plugin/lib/cache.ml

```diff
@@ -39,6 +38,10 @@ module Digests = struct
   let project (mk_digest : namespace:string -> digest) : digest =
     mk_digest ~namespace:"project"
 
+  (* Creates a digest for the program state cache. *)
+  let program (mk_digest : namespace:string -> digest) : digest =
+    mk_digest ~namespace:"program"
```

Comment:
Is this at all related to Ivan's suggestion of namespacing or is this a separate thing?

Reply:
I removed the package name since we aren't storing the program in the knowledge base anymore. This is used to create the digest for the program cache, which is separate.